spectral_cube/io/fits.py

```diff
@@ -226,10 +234,7 @@ def write_fits_cube(cube, filename, overwrite=False,
         hdulist[0].header.add_history("Written by spectral_cube v{version} on "
                                       "{date}".format(version=SPECTRAL_CUBE_VERSION,
                                                       date=now))
-        try:
-            hdulist.writeto(filename, overwrite=overwrite)
-        except TypeError:
-            hdulist.writeto(filename, clobber=overwrite)
+        hdulist.writeto(filename)
```

Comment:
```suggestion
        hdulist.writeto(filename, overwrite=overwrite)
```

Reply:
@keflavich -- `include_origin_notes` has always been unused here. Is this still a to-do or could it be removed?

Reply:
oh.  That should make this line: https://github.com/radio-astro-tools/spectral-cube/pull/618/files#diff-fa807408d14cd8bb4abf0ef50eba4a31R234 optional

Reply:
Fixed